tsconfig.json

```diff
@@ -10,12 +10,10 @@
     "strictNullChecks": true,
     "target": "ES5",
     "types": [
-      "env",
       "mocha"
     ],
     "typeRoots": [
-      "node_modules/@types",
-      "typings"
```

Comment:
What do these changes do? Sorry, I need to learn more about TypeScript...

Reply:
These are the paths to the typed definitions. 

So, if you are importing plain javascript, this will look into these directories to see if there are any definitions available. If none are available, the compiler will complain because it doesn't know what the parameter or return types are.

It's kind of equivalent to pointing to the `includes` directory for a C compiler and referencing the `.h` files.